### PR TITLE
ci: bring back `aarch64-unknown-linux-musl`

### PR DESCRIPTION
**Pull request by quad**
This reverts commit 841fb0b7afc2df8079986eb3442f278188e033b2.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -121,9 +121,8 @@ jobs:
             binary: sccache-dist
             extra_args: --no-default-features --features="dist-server"
             target: x86_64-unknown-linux-musl
-# Disable because E: Unable to locate package musl-dev:arm64
-#          - os: ubuntu-20.04
-#            target: aarch64-unknown-linux-musl
+          - os: ubuntu-20.04
+            target: aarch64-unknown-linux-musl
```
[the remark this answers is not in the document]
Comment (Xuanwo) [reply .github/workflows/ci.yml:125]: Is it a good idea to use ubuntu-22.04 to build our arm64 release? cc @sylvestre

[the remark this answers is not in the document]
Comment (sylvestre) [reply .github/workflows/ci.yml:125]: Sure, no worries

[the remark this answers is not in the document]
Comment (Xuanwo) [reply .github/workflows/ci.yml:125]: @quad, Do you wanna to give it a try?

[the remark this answers is not in the document]
Comment (quad) [reply .github/workflows/ci.yml:125]: I'm already fixing it to use ports. I'm waiting for a confirmation build now.

**Comment by quad** [comment]
@sylvestre @Xuanwo This PR is ready to go! 🚀

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Xuanwo) [reply .github/workflows/ci.yml:125]: > But … 22.04 ain't a bad idea. I can make another PR for that while this one bakes?

LGTM!

**Review by Xuanwo — APPROVED** [review]
Thanks!

**Comment by sylvestre** [comment]
I will merge it in the next Few hours and make a new release

**Comment by quad** [comment]
@sylvestre I eagerly await a new release! Unsurprisingly, we use the arm64 builds, so we're stuck on 0.5.4 until it's out! 🙏

**Pull request merged**